src/felleskomponenter/textarea/Textarea.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import { autobind } from 'nav-frontend-js-utils';
+import classNames from 'classnames';
+import SkjemaelementFeilmelding from 'nav-frontend-skjema/lib/skjemaelement-feilmelding';
+import { SkjemaelementFeil } from 'nav-frontend-skjema/lib/skjemaelement-feilmelding';
+import 'nav-frontend-skjema-style';
+import './Textarea.less';
+
+const inputCls = (className: string, harFeil: SkjemaelementFeil | undefined) =>
+    classNames(className, 'skjemaelement__input textarea--medMeta', 'textarea-custom', {
+        'skjemaelement__input--harFeil': harFeil
+    });
+
+export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+    label: React.ReactNode;
+    maxLength: number;
+    value: string;
+    textareaClass: string;
+    id: string;
+    name: string;
+    onChange: (event: React.SyntheticEvent<EventTarget>) => void;
+    tellerTekst: (antallTegn: number, maxLength: number) => React.ReactNode;
+    feil?: SkjemaelementFeil;
+}
+
+class Textarea extends React.Component<TextareaProps> {
+    private tekstomrade: HTMLTextAreaElement | null = null;
+
+    constructor(props: TextareaProps) {
+        super(props);
+        autobind(this);
+    }
+
+    onChange(event: React.SyntheticEvent<EventTarget>) {
+        if (!!this.tekstomrade) {
+            this.tekstomrade.style.height = 'inherit';
+            this.tekstomrade.style.height = this.tekstomrade.scrollHeight + 'px';
+        }
+
+        this.props.onChange(event);
+    }
+
+    render() {
+        const { label, maxLength, textareaClass, id, feil, tellerTekst, ...other } = this.props;
```

Comment:
Er ikke så fan av å bruke spread sånn og så sende det avgårde til en annen komponent. Selv om `props` er typa, så er det vanskelig å lese fordi jeg må scrolle opp og ned for å "regne ut" hva det er som egentlig sendes.